pkg/workloadmeta/types.go

```diff
@@ -698,6 +703,60 @@ func (n KubernetesNode) String(verbose bool) string {
 
 var _ Entity = &KubernetesNode{}
 
+// KubernetesDeployment is an Entity representing a Kubernetes Deployment.
+type KubernetesDeployment struct {
+	EntityID
+	EntityMeta
+	LanguagesForContainer map[EntityID][]languagemodels.Language
```

Comment:
Actually, I am not sure we need this because we'll store the information in the annotations